beta/src/components/MDX/Sandpack/SandpackRoot.tsx

```diff
@@ -77,15 +77,19 @@ function SandpackRoot(props: SandpackProps) {
   };
 
   return (
-    <div className="sandpack-container my-8" translate="no">
+    <div className="sandpack sandpack--playground sandbox my-8">
       <SandpackProvider
         template="react"
-        customSetup={{...setup, files: files}}
-        autorun={autorun}
-        initMode="user-visible"
-        initModeObserverOptions={{rootMargin: '1400px 0px'}}
-        bundlerURL="https://6b760a26.sandpack-bundler.pages.dev"
-        logLevel={SandpackLogLevel.None}>
+        files={files}
+        customSetup={setup}
+        theme={CustomTheme}
+        options={{
+          autorun,
+          initMode: 'user-visible',
+          initModeObserverOptions: {rootMargin: '1400px 0px'},
+          bundlerURL: 'https://6b760a26.sandpack-bundler.pages.dev',
```

Comment:
I didn't update the bundler in order to avoid unnecessary changes, but I'd recommend upgrading it later.